fast_llm/data/preparator/hf_processors/implementations/agregator.py

```diff
@@ -0,0 +1,11 @@
+import datasets
+
+from fast_llm.data.preparator.hf_processors.configs.agregator import AgregatorConfig
+
+def apply(config: AgregatorConfig, dataset: datasets.Dataset) -> datasets.Dataset:
```

Comment:
This won't work with import by name, you'll need a more unique name. Also consider making it a (near-trivial) `Configurable` class.